docs/src/authorizations_processors/callbacks.md

```diff
@@ -8,6 +8,8 @@ For the execution of `NonAtomic` batches, each function in the batch can optiona
 
 If the processor receives the expected callback from the correct address, the batch will move to the next function. If it receives a different callback than expected from that address, the execution of that function will be considered failed and it will be retried (if applicable). In any case, a callback must be received to determine if the function was successful or not.
```

Comment:
```suggestion
If the processor receives the expected callback from the correct address, the batch will move to the next function. If it receives a different callback than expected from that address, the execution of that function is considered to have failed and it will be retried (if applicable). In either case, a callback must be received to determine if the function was successful or not.
```